tests/behat/bootstrap/DbTrait.php

```diff
@@ -109,4 +109,16 @@ public function IDeleteTheAppSetting($setting)
         $command = sprintf('psql %s -c "%s"', self::$dbName, $query);
         exec($command);
     }
+
+    /**
+     * @Given I discharge the deputies from case :caseNumber
+     */
+    public function iDischargeTheDeputiesFromCase($caseNumber)
+    {
+//        $query = "DELETE FROM deputy_case where client_id = (SELECT id FROM client WHERE case_number = '{$caseNumber}'";
```

Comment:
Should remove this `DELETE FROM` comment to avoid confusion

Reply:
removed